plugins/techdocs/src/home/components/OverviewContent.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { generatePath } from 'react-router-dom';
+import { Grid } from '@material-ui/core';
+
+import { Entity } from '@backstage/catalog-model';
+import {
+  Content,
+  ContentHeader,
+  SupportButton,
+  ItemCard,
+} from '@backstage/core';
+
+import { rootDocsRouteRef } from '../../plugin';
+
+export const OverviewContent = ({ value }: { value: Entity[] | undefined }) => {
+  if (!value) return null;
+  return (
+    <Content>
+      <ContentHeader
+        title="Overview"
+        description="Explore your internal technical ecosystem through documentation."
+      >
+        <SupportButton>Discover documentation in your ecosystem.</SupportButton>
```

Comment:
I was reading #4102 and it says 
> a Support Button should exist with the possibility for a Backstage integrator to insert information to the team responsible for TechDocs within the organisation

The support button here looks static without the possibility to change things like the `slackChannel`. I think it's inline with some other support buttons from other plugins, however it would be great if the support information could be injected into the component.

I point this out as while implementing backstage at my organisation I found that to modify things like the support button I was having to recreate whole pages and we miss out on great improvements like this owned docs tab. Unless of course I am missing a way to do this/this is the intended pattern.

Would like to know your thoughts, if this is something that could be included perhaps in this PR, or I can raise an issue if I've misunderstood #4102

Reply:
We need to figure out a setup to let Backstage integrators configure "Support" information, not just for TechDocs but for all the plugins. Doing it via `app-config.yaml` is likely to be discouraged since the file is going to get inflated - this is likely to be done by code when a Backstage app is setup.

cc @backstage/maintainers

Reply:
Related https://github.com/backstage/backstage/pull/4517

`app.support` in `app-config.yaml` might be a good start.